### PR TITLE
feat: reviewed and simplified the node status check

### PR DESCRIPTION
**Pull request by vojtechsimetka**
Reviewed the wording and refactored to eliminate un-renderable states but also to make the node setup a bit cleaner. This can be even further simplified when we have a context provider (see #40 ) and we can even store state there.

### CODE REVIEW

File: src/pages/status/SetupSteps/ChequebookDeployFund.tsx [hunk 1]
```diff
@@ -1,38 +1,22 @@
...
+const ChequebookDeployFund = ({ isLoading, chequebookAddress, chequebookBalance }: Props): ReactElement | null => {
+  if (isLoading) return null
+
+  return (
+    <div>
+      <p style={{ marginBottom: '20px', display: 'flex' }}>
+        {chequebookAddress?.chequebookaddress && <DepositModal />}
+      </p>
+      <div style={{ marginBottom: '10px' }}>
+        {!(chequebookAddress?.chequebookaddress && chequebookBalance && chequebookBalance?.totalBalance > 0) && (
```
Comment (agazso) [inline src/pages/status/SetupSteps/ChequebookDeployFund.tsx:18]: I find this logical condition a bit hard to follow. Maybe it would be worth extracting some values to variables to simplify.

**Review by agazso — APPROVED** [review]
Looks good, but I haven't tested 😄

File: src/pages/status/SetupSteps/ChequebookDeployFund.tsx (hunk 1 above)
Comment (vojtechsimetka) [reply src/pages/status/SetupSteps/ChequebookDeployFund.tsx:18]: Yeah these should be simple components that just display values that are always defined. But such rewrite seemed beyond the scope what we are doing now :)
